### ISSUE TITLE
Test program (test.py) not genering any output

### ISSUE THREAD

**Issue by amujeeb99** [khalooei/ALOCC-CVPR2018#18]
Issue:    No output sample is generated for “mnist” or “ucsd” (even when the pre-trained model provided in github is used)

We used following commands (separately) to run the test program on the dataset provided and using the pre-trained model provided as per the instructions. Neither "mnist" or "ucsd" output images are generated. The code fragment is copy-pasted below.
==================
# for test on UCSD and patch_size 45*45 and some specific dir like ['Test004'], etc. We prefer to open test.py file and edit every thing that you want
python test.py --dataset UCSD --dataset_address ./dataset/UCSD_Anomaly_Dataset.v1p2/UCSDped2/Train --input_height 45 --output_height 45 

# for test on MNIST
python test.py --dataset mnist --dataset_address ./dataset/mnist/ --input_height 28 --output_height 28
================

Following is the copy-paste of the code in test.py file which is executed (after some bug fixing)
=======================================
if FLAGS.dataset=='mnist':
            mnist = input_data.read_data_sets(FLAGS.dataset_address)

            specific_idx_anomaly = np.where(mnist.train.labels != 6)[0]
            specific_idx = np.where(mnist.train.labels == 6)[0]
            ten_precent_anomaly = [specific_idx_anomaly[x] for x in
                                   random.sample(range(0, len(specific_idx_anomaly)), len(specific_idx) // 40)]

            data = mnist.train.images[specific_idx].reshape(-1, 28, 28, 1)
            tmp_data = mnist.train.images[ten_precent_anomaly].reshape(-1, 28, 28, 1)
            data = np.append(data, tmp_data).reshape(-1, 28, 28, 1)

            lst_prob = tmp_ALOCC_model.f_test_frozen_model(data[0:FLAGS.batch_size])
            print('check is ok')
            exit()

# else in UCDS (depends on infrustructure)
        for s_image_dirs in sorted(glob(os.path.join(FLAGS.dataset_address, 'Test[0-9][0-9][0-9]'))):
            tmp_lst_image_paths = []
            
            if os.path.basename(s_image_dirs) not in ['Test004']:
               print('Skip ',os.path.basename(s_image_dirs))
               continue
            for s_image_dir_files in sorted(glob(os.path.join(s_image_dirs + '/*'))):
                if os.path.basename(s_image_dir_files) not in ['068.tif']:
                    print('Skip ', os.path.basename(s_image_dir_files))
                    continue
                tmp_lst_image_paths.append(s_image_dir_files)

            #random
            #lst_image_paths = [tmp_lst_image_paths[x] for x in random.sample(range(0, len(tmp_lst_image_paths)), n_fetch_data)]
            lst_image_paths = tmp_lst_image_paths
            #images =read_lst_images(lst_image_paths,nd_patch_size,nd_patch_step,b_work_on_patch=False)
            
            nd_patch_step = 0              # we added this line (from the github to avoid an error)
           images = read_lst_images_w_noise2(lst_image_paths, nd_patch_size, nd_patch_step)

            lst_prob = process_frame(os.path.basename(s_image_dirs),images,tmp_ALOCC_model)
            print('pseudocode test is finished')

**Comment by khalooei** [comment]
Dear @amujeeb99 , Welcome to follow the ALOCC research project. We hope this idea could help your research plan or your business, product as previous researchers. But I didn't get what you requested in this issue, Do you mean that you can't generate any output image or you mean you can finally solve your latest questions with that following code as you mentioned?

**Comment by amujeeb99** [comment]
Dear Khalooei,
Yes, we are not able to generate the output image file (when we run test.py). Either for UCSD or mnist. We have tried with the pre-trained model files provided with the code.

**Comment by khalooei** [comment]
> Dear Khalooei,
> Yes, we are not able to generate the output image file (when we run test.py). Either for UCSD or mnist. We have tried with the pre-trained model files provided with the code.

Dear @amujeeb99, please send your logs or output, which you see when using the standard command which I said in the readme.
